modules/aws-emr-cluster/outputs.tf

```
@@ -17,9 +17,34 @@ output "release_label" {
   description = "The release label for the Amazon EMR release."
 }
 
-output "core_group_instance_count" {
-  value       = var.core_group_instance_count
-  description = "Number of cores configured to execute the job flow"
+output "master_instance_on_demand_count" {
+  value       = var.master_instance_on_demand_count
+  description = "Number of on-demand master instances configured to execute the job flow"
+}
+
+output "master_instance_spot_count" {
+  value       = var.master_instance_spot_count
+  description = "Number of spot master instances configured to execute the job flow"
```

Nit: I think without the "configured to execute the job flow" we convey the same information.
```suggestion
  description = "Number of spot master instances"
```